### ISSUE TITLE
Schema validation: Property 'tags' is not allowed 

### ISSUE THREAD

**Issue by marcelstoer** [asciidoctor/asciidoctor-intellij-plugin#646]
## Observed vs. expected behavior

The plugin thinks `tags` be not an allowed key of the `sources` key in `antora-playbook.yml` - which it obviously is as per https://docs.antora.org/antora/2.3/playbook/configure-content-sources/#sources-key

I had a brief look at https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/master/src/main/resources/jsonSchemas/antoraPlaybookSchema.json but I don't see anything wrong with it at first glance.

## Steps to reproduce

```yaml
site:
  title: "demo"
content:
  edit_url: "{web_url}/edit/master/{path}"
  sources:
    - url: https://github.com/asciidoctor/asciidoctor-intellij-plugin
      start_path: doc
      tags: ['*.*.*']
ui:
  bundle:
    url: https://gitlab.com/antora/antora-ui-default/-/jobs/artifacts/master/raw/build/ui-bundle.zip?job=bundle-stable
    snapshot: true
```
<img width="450" alt="Screen Shot 2021-01-28 at 16 56 22" src="https://user-images.githubusercontent.com/624195/106164001-d2347880-6189-11eb-8e70-91d4b8e2ca92.png">

## Environment

Plugin Version: 0.32.12

IntelliJ Details: 
IntelliJ IDEA 2020.3.2 (Ultimate Edition)
Build #IU-203.7148.57, built on January 26, 2021
Licensed to Marcel Stör
Subscription is active until October 16, 2021.
Runtime version: 11.0.9.1+11-b1145.77 x86_64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
macOS 10.16
GC: ParNew, ConcurrentMarkSweep
Memory: 1979M
Cores: 16
Registry: ide.mac.hide.cursor.when.typing=false, debugger.watches.in.variables=false, editor.zero.latency.rendering=false
Non-Bundled Plugins: com.intellij.apacheConfig, com.jetbrains.plugins.ini4idea, org.zalando.intellij.swagger, com.tang, org.jetbrains.plugins.hocon, MavenRunHelper, com.dmarcotte.handlebars, org.jetbrains.kotlin, com.intellij.plugins.html.instantEditing, com.jetbrains.php, PlantUML integration, Pythonid, org.asciidoctor.intellij.asciidoc

**Comment by ahus1** [comment]
Thanks for reporting this issue. I double-checked and found that the Schema needs to have two "tags" properties, one for content (to override the default), and one for each source. The "tags" property for the source was missing. 

I published pre-release 0.32.13 that should fix this. Please give it a try.

The pre-release of the plugin is available from [GitHub releases](https://github.com/asciidoctor/asciidoctor-intellij-plugin/releases) and the [IntelliJ AsciiDoc EAP repository](https://intellij-asciidoc-plugin.ahus1.de/docs/contributors-guide/user/add-eap-repository-to-ide.html).

**Comment by marcelstoer** [comment]
That was quick, thanks!

Bummer, at the first glance I missed that it was missing for the `sources` key. Once you look at the diff becomes painfully obvious 😞 

Maybe you know for sure but I don't think `start_paths`, `branches`, and `tags` can be strings; they have to be arrays AFAIK (wouldn't otherwise make much sense to use the plural form). Do you have tests for those?

**Comment by mojavelinux** [comment]
> Maybe you know for sure but I don't think start_paths, branches, and tags can be strings

That is correct. A single string value is recognized and handled accordingly by the application.

**Comment by marcelstoer** [comment]
Now I'm confused. The JSON schema here says that there's [`start_path` as string](https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/master/src/main/resources/jsonSchemas/antoraPlaybookSchema.json#L252) plus the plural [`start_paths` that can be an array or a string](https://github.com/asciidoctor/asciidoctor-intellij-plugin/blob/master/src/main/resources/jsonSchemas/antoraPlaybookSchema.json#L256) - that felt too odd to be correct 😉 I double checked the documentation at https://docs.antora.org/antora/2.3/playbook/configure-content-sources/#sources-key but didn't find a more definite answer to that. But I did realize https://docs.antora.org/antora/2.3/playbook/configure-content-sources/#sources-reference is missing `sources[n].branches` and `sources[n].tags`.

**Comment by ahus1** [comment]
I suppose the truth is in Antora's code. It'll be some days (maybe more) before I find the time to dig into it. The schema you see today was created from the schema parser in Antora.

Any help is appreciated - if you notice other discrepancies with Antora in action or in the code, please join in.

Once we see the truth in the code we can align the schema and the docs.

**Comment by mojavelinux** [comment]
> The JSON schema here says that there's start_path as string plus the plural start_paths that can be an array or a string - that felt too odd to be correct

Well, it's correct. start_path is singular and thus implies only one. start_paths is plural and is thus a collection, though pretty much anywhere in Antora's config where multiple values are allowed we coerce a single value into a collection.

**Comment by mojavelinux** [comment]
> But I did realize docs.antora.org/antora/2.3/playbook/configure-content-sources/#sources-reference is missing sources[n].branches and sources[n].tags.

Oops, that table is incomplete. Can you file an issue?

**Comment by marcelstoer** [comment]
Fixed with 72cb38fd05c5e5ce68cfe6f99debf1be316b22da and available with 0.32.13 -> closing

> Oops, that table is incomplete. Can you file an issue?

I tried but failed: https://gitlab.com/antora/antora/-/merge_requests/612

**Comment by mojavelinux** [comment]
Thanks! (Don't worry about the build failure. It is not related to the docs change).
